e2e/bittwister/suite_setup_test.go

```diff
@@ -16,11 +17,11 @@ type Suite struct {
 }
 
 func (s *Suite) SetupSuite() {
-	var (
-		err error
-		ctx = context.Background()
-	)
-	s.Knuu, err = knuu.New(ctx, knuu.Options{ProxyEnabled: true})
+	ctx := context.Background()
+	k8sClient, err := k8s.NewClient(ctx, knuu.DefaultTestScope())
+	s.Require().NoError(err)
```

Comment:
if we revert your change of adding the k8 client as a top level argument and removing it from the `setdefaults` function, then you could revert this.